src/doc/man/cargo-package.md

```diff
@@ -81,8 +81,45 @@ There is no guarantee that the source code in the tarball matches the VCS inform
 
 {{#options}}
 
-{{#option "`-l`" "`--list`" }}
+{{#option "`-l`" "`--list`_[fmt]_" }}
 Print files included in a package without making one.
+
+Specifying an output format (rather than the default) is unstable and requires
+`-Zunstable-options`. Valid output formats:
+
+- `json` (unstable, requires `-Zunstable-options`): Emit machine-readable JSON
+  information about each file.
+  ```javascript
+  {
+    /* The Package ID Spec of the package. */
+    "path+file:///home/foo#0.0.0": {
+      /* Relative path in the archive file. */
+      "Cargo.toml.orig": {
+        /* An absolute path to the actual file content. */
+        "original_path": "/home/foo/Cargo.toml",
```

Comment:
snake case or kebab case?

Reply:
We do that i `-Zsbom`, `cargo metadata`, and [`--message-format=json`](https://doc.rust-lang.org/nightly/cargo/reference/external-tools.html#compiler-messages). This follows.

Reply:
sbom is just because of `cargo metadata`.  We deferred what it should actually do to the RFC. (I brought this up there).

Uck, didn't know `--message-format=json` does that.

 My hope for the new plumbing commands is that we reuse public schemas as much as possible, like `TomlManifest`.  Unless we copy/paste it, that will be kebab case.

So for right now, our [schema document](https://doc.crates.io/contrib/implementation/schemas.html#schema-design) gives an unqualified statement to use kebab case.  We should probably discuss this and figure out if there needs to be more qualifiers to it.